### PR TITLE
Add qt-quick3d

### PR DESCRIPTION
**Pull request by jschueller**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

cc @hmaarrfk

### CODE REVIEW

File: recipes/qt-quick3d/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  summary: Qt is a cross-platform application and UI framework.
+  description: |
+    Qt helps you create connected devices, UIs & applications that run
+    anywhere on any device, on any operating system at any time.
+  doc_url: http://doc.qt.io/
+  dev_url: https://github.com/qt/qtquick3d
+
+extra:
+  recipe-maintainers:
+    - jschueller
```
Comment (hmaarrfk) [inline recipes/qt-quick3d/meta.yaml:72]:
```suggestion
    - jschueller
    - conda-forge/qt-main
```

**Comment by hmaarrfk** [comment]
I take it you agree with the transition plan to do this on qt6 6.7.2? 

It just seems that the ETA for that one is quite late....

**Comment by jschueller** [comment]
yes, I just updated qt 6.7.2

**Comment by hmaarrfk** [comment]
oh wow. thank you for that! you are so quick...

**Comment by hmaarrfk** [comment]
Well FWIW, qt6-main dropped in size by about 7 MB. So thank you!
image

**Review by hmaarrfk — APPROVED** [review]

**Comment by jschueller** [comment]
cc @xhochy @carterbox

**Comment by hmaarrfk** [comment]
@jschueller do you really want one of them review? or can i click merge?

**Comment by jschueller** [comment]
yes, please do, didnt know you could merge

**Pull request merged**